spec.html

```diff
@@ -471,8 +471,24 @@ <h1>The Syntactic Grammar</h1>
-      <p>Nonterminal symbols are shown in <i>italic</i> type. The definition of a nonterminal (also called a &ldquo;production&rdquo;) is introduced by the name of the nonterminal being defined followed by one or more colons. (The number of colons indicates to which grammar the production belongs.) One or more alternative right-hand sides for the nonterminal then follow on succeeding lines. For example, the syntactic definition:</p>
+      <p>The nonterminal symbols of all the ECMAScript grammars are shown in <i>italic</i> type.</p>
+      <p>In all the grammars, 'verbatim' terminal symbols are shown in `fixed-width` font. These are to appear in a source text exactly as written. All terminal symbol code points specified in this way are to be understood as the appropriate Unicode code points from the Basic Latin range, as opposed to any similar-looking code points from other Unicode ranges.</p>
+      <p>In grammars whose terminal symbols are individual Unicode code points (i.e., the lexical, RegExp, and numeric string grammars), a contiguous run of multiple fixed-width code points appearing in a production is a simple shorthand for the same sequence of code points, written as standalone terminal symbols.</p>
+      <p>For example, the production:</p>
+      <emu-grammar type="example">
+        HexIntegerLiteral :: `0x` HexDigits
+      </emu-grammar>
+      <p>is a shorthand for:</p>
+      <emu-grammar type="example">
+        HexIntegerLiteral :: `0` `x` HexDigits
+      </emu-grammar>
+      <p>Conversely, in the syntactic grammar, a contiguous run of fixed-width code points is a single terminal symbol.</p>
+      <p>Terminal symbols come in two other forms:</p>
+      <ul>
+        <li>In the lexical and RegExp grammars, Unicode code points without a conventional printed representation are instead shown in the form "&lt;FOO&gt;" where "FOO" is a mnemonic for the code point. These forms are defined in <emu-xref href="#sec-unicode-format-control-characters" title></emu-xref> and <emu-xref href="#sec-white-space" title></emu-xref>.</li>
+        <li>In the syntactic grammar, certain terminal symbols (e.g. |IdentifierName| and |RegularExpressionLiteral|) are shown in italics, as they refer to the nonterminals of the same name in the lexical grammar.</li>
+      </ul>
+      <p>The definition of a nonterminal (also called a &ldquo;production&rdquo;) is introduced by the name of the nonterminal being defined followed by one or more colons. (The number of colons indicates to which grammar the production belongs.) One or more alternative right-hand sides for the nonterminal then follow on succeeding lines. For example, the syntactic definition:</p>
```

Comment:
> (The number of colons indicates to which grammar the production belongs.)

I would take this out of the parenthetical.

Reply:
My PR is only about symbols, not productions, so that text isn't part of it. The line is only diff-marked because I extracted the first sentence from that paragraph.

Reply:
I see, but I still think the parentheses are unnecessary. Do you think that needs to be in a separate PR?

Reply:
Yup.

It'd be in-scope if I were overhauling the whole clause, but I wisely stopped well short of that.